bridge-adaptivity/bridge_lti/consumer.py

```diff
@@ -25,30 +26,41 @@ def tool_config(request):
     return HttpResponse(lti_tool_config.to_xml(), content_type='text/xml')
 
 
-def content_source(request, pk):
+def source_preview(request):
     """
-    Simple view to render Source's component shared through LTI.
+    Simple view to render content Source's block shared through LTI.
     """
-    lti_source = LtiSource.objects.get(id=pk)
+    activity_id = request.GET.get('activity_id')
+    if activity_id:
+        activity = Activity.objects.get(id=activity_id)
```

Comment:
Do we need to handle the exception in case Activity with actitity_id won't be found?

Reply:
No, it should exist. Critical error otherwise.